src/lib/authserver/cli/auth-cli.rb

```diff
@@ -0,0 +1,23 @@
+Yast.import "CommandLine"
+
+module AuthServer
+  class CLI
+    extend Yast::I18n
+
+    def self.run(cmd="authserver")
+      textdomain "authserver"
+
+      msg = format(
+        _("Command line is not supported. Use '%s' directly."),
```

Comment:
Is there really ldap-server and krb-server CLI commands? I do not see it. If you just do not support CLI at all and there there is no good replacement, you can omit that "help" parameter and it will just write CLI not supported.

Reply:
Sorry I don't understand this comment. Can you explain what you mean here?

Reply:
Well, what I mean here. That this help text is showed when user print some command line arguments so e.g.:
```
yast2 auth-cli help
```
say use `Use 'auth-cli' directly.` but such command does not exists. We use it often to provide existing CLI tool that user can use e.g. for firewall we recommend `firewall-cli` to use. But if I get it here is not direct replacement for functionality that auth-cli provide that works on command line. So in that case I suggest to use solution we use e.g. in storage-ng where is also no suitable CLI tool and just say CLI not available - https://github.com/yast/yast-storage-ng/pull/1170#discussion_r527675569

so simply you can remove that msg and "help" key from map and it will print just "CLI not supported"